source/utils/environment.js

```diff
@@ -1,4 +1,4 @@
-import { get } from 'lodash';
+import get from 'lodash/get';
```

Comment:
Hmm, I thought you just merged this change?

Reply:
Just checked master and it contains this change. However, I don't think the rebuilt bundle was included.

Reply:
Can you merge master into this branch?

Reply:
Would appear this PR is unnecessary.